src/src/models/SelectModel.ts

```diff
@@ -0,0 +1,18 @@
+import OptionModel from "./OptionModel";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
import OptionModel from './OptionModel';
```